include/seqan3/range/view/detail.hpp

```diff
@@ -322,6 +317,79 @@ class generic_pipable_view_adaptor : public pipable_adaptor_base<generic_pipable
     {
         return view_type{std::forward<arg_types>(args)...};
     }
+
+public:
+    //!\brief Inherit the base type's constructors.
+    using base_type::base_type;
+};
+
+// ============================================================================
+//  adaptor_from_functor
+// ============================================================================
+
+/*!\brief Template for range adaptor closure objects that store arguments and wrap a proto-adaptor.
```

Comment:
maybe include a proto-adaptor definition in the tutorial since you reference this class in the tutorial anyway...

so 
```
                    adaptor closure object 
                  /
adaptor object --- 
                  \
                    proto adaptor
```
?

Reply:
so proto adaptor was dropped? Is there another word for it? because it is confusing if only we use it now.

Reply:
I will do some changes to the toturials next week and include something like this.

Reply:
(yes proto-adaptor as a term was dropped from the standard, there is no special term for them anymore, but we can still introduce it in our documentation)

Reply:
ok fine. but then add a definition in the tutorial and the documentation.